features/mesh/dataplanes/Index.feature

```diff
@@ -68,9 +79,52 @@ Feature: mesh / dataplanes / index
   Scenario: The Proxy listing has the expected content and UI elements
     When I visit the "/meshes/default/data-planes" URL
 
-    Then the "$table-row" element exists 9 times
-    Then the "$table-row:nth-child(1)" element contains
+    Then the "$table tbody tr" element exists 9 times
+    Then the "$item" element contains
```

Comment:
bit of a nit but I'd like to avoid CSS selectors inside the tests where possible and use aliases (our aliases are almost pageObject-y and also/therefore read better)

I think this test was one of those early ones but it looks like in other tests we have

"$item" = tbody tr

```gherkin
Then the "$item" element exists 9 times
Then the "$item:nth-child(1)" element contains
```

Then if I need to I can do:
```gherkin
Then the "$item:nth-child(3)" element contains...
Then the "$item:nth-child(7)" element contains...
Then the "$item:nth-child(100)" element contains
```

..if I ever need to.

Would be good to follow that pattern and change up where we don't as we go, unless there is a reason not to.

Reply:
I’ve changed this.